### PR TITLE
OCLOMRS-919: Added ability to sort Answers via drag and drop

### PR DESCRIPTION
**Pull request by nk183**
# JIRA TICKET NAME:
[OCLOMRS-919](https://issues.openmrs.org/browse/OCLOMRS-919)

# Summary:
Uses `drag n drop ` in order to re-order the AnswerMapping 
added an `extras` property in the Mapping class which stores `{sort_wieght: number}` which will update when after re-ordering, we click submit and be only available to `Custom Created Concept` on the `edit mode` of concept

### CODE REVIEW

**Comment by nk183** [comment]
@ibacher please review this PR

**Comment by nk183** [comment]
@ibacher Now everything is fine

File: src/components/drag_handle-icon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" viewBox="0 0 24 24" fill="black" width="18px" height="18px"><g><rect fill="none" height="24" width="24"/></g><g><g><g><path d="M20,9H4v2h16V9z M4,15h16v-2H4V15z"/></g></g></g></svg>
```
Comment (ibacher) [inline src/components/drag_handle-icon.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" viewBox="0 0 24 24" fill="black" width="18px" height="18px"><g><rect fill="none" height="24" width="24"/></g><g><g><g><path d="M20,9H4v2h16V9z M4,15h16v-2H4V15z"/></g></g></g></svg>

```

File: tsconfig.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "strict": true,
 //    "noUnusedLocals": true,  // todo, we'd like to make stuff even more strict, but pausing this work for the moment
     "forceConsistentCasingInFileNames": true,
+    "noImplicitAny": false,
```
Comment (ibacher) [inline tsconfig.json:16]:
```suggestion
```

**Review by ibacher — APPROVED** [review]

File: extras.patch [hunk 3]
```diff
@@ -0,0 +1,141 @@
+diff --git a/src/apps/concepts/components/ConceptForm.tsx b/src/apps/concepts/components/ConceptForm.tsx
```
[the remark this answers is not in the document]
Comment (nk183) [reply extras.patch]: I have removed it

**Pull request merged**